ui/packages/tailwind-config/tailwind.config.js

```diff
@@ -169,15 +169,15 @@ module.exports = {
             mono: ['Menlo', 'Monaco', 'Consolas', 'Liberation Mono', 'Courier New', 'monospace'],
         },
         fontSize: {
-            '2xs': remCalc('9'),
-            xs: remCalc('10'),
-            sm: remCalc('11'),
-            base: remCalc('12'),
-            lg: remCalc('13'),
-            xl: remCalc('14'),
-            '2xl': remCalc('16'),
-            '3xl': remCalc('20'),
-            '4xl': remCalc('24'),
```

Comment:
Does it make more sense
* leave these alone
* in ui/apps/platform/tailwind.config.js import and then override `fontSize` property?

```js
module.exports = require('@stackrox/tailwind-config');
```